### PR TITLE
PEAR-1716: add focus styles to the buttons in blue background for now

### PR DESCRIPTION
**Pull request by paribartandhakal**
## Description
We are doing it this way for now as not to introduce any side effects to other buttons. This will be taken care in https://gdc-ctds.atlassian.net/browse/PEAR-1730 after the soft launch.
cc. @craigrbarnes @cemile-ctds @wteouchicago 

## Checklist
- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

File: packages/portal-proto/tailwind.config.js [hunk 1]
```diff
@@ -768,6 +768,7 @@ module.exports = {
           8: "#7A7A15",
           9: "#10828E",
         },
+        focusColor: "rgb(39, 93, 197)",
```
Comment (paribartandhakal) [inline packages/portal-proto/tailwind.config.js]: likely to be changed

File: packages/portal-proto/src/utils/index.ts [hunk 2]
```diff
@@ -289,3 +289,6 @@ export const getAnnotationsLinkParamsFromFiles = (
     file.annotations.map((annotation) => `"${annotation.annotation_id}"`),
   ]}]},"op":"in"}],"op":"and"}`;
 };
+
+export const focusStyles =
+  "focus-visible:outline-none focus-visible:ring-offset-2 focus:ring-offset-white rounded-md focus-visible:ring-inset focus-visible:ring-2 focus-visible:ring-focusColor";
```
Comment (paribartandhakal) [inline packages/portal-proto/src/utils/index.ts:294]: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible

**Comment by wteouchicago** [comment]
I defer to @cemile-ctds for checks on this one.

**Comment by cemile-ctds** [comment]
This looks good on my side.

File: packages/portal-proto/tailwind.config.js (hunk 1 above)
Comment (paribartandhakal) [reply packages/portal-proto/tailwind.config.js]: updated!

**Comment by paribartandhakal** [comment]
The PR has been updated with the right focus ring color. Good to review now!

**Review by cemile-ctds — APPROVED** [review]

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**